project.clj

```diff
@@ -3,5 +3,12 @@
   :url "https://github.com/weavejester/suspendable"
   :license {:name "The MIT License"
             :url "http://opensource.org/licenses/MIT"}
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [com.stuartsierra/component "0.2.3"]])
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [com.stuartsierra/component "0.2.3"]]
+  :profiles {:cljs {:dependencies [[org.clojure/clojurescript "1.10.597"]
+                                   [org.clojure/clojure "1.10.1"]
+                                   [com.stuartsierra/component "0.4.0"]]}}
```

Comment:
Note:
* ClojureScript support is not in `component "0.2.3"`
* ClojureScript `1.10.597` doesn't work with `clojure "1.7.0"`.

I decided to put all the latest dependencies in the `cljs` profile instead of updating the default profile's dependencies to avoid affecting anyone's dependency tree upstream.

I'm open to suggestions on if you would have done anything differently.

Reply:
Can you update component to `"0.4.0"` globally, and then put the ClojureScript dependency in the `:provided` profile? What's the minimum Clojure version that's needed by ClojureScript?

Reply:
Updated component to `"0.4.0"` and clojure only had to be bumped to `"1.8.0"` for the cljs tests to pass, so I updated clojure to that version.

I also renamed the `:cljs` profile to `:provided`. I wasn't sure if this was what you meant or to put `:scope "provided"` on a top-level clojurescript dependency.